### PR TITLE
[WebProfilerBundle] Tweak Mailer panel rendering

### PR DESCRIPTION
**Pull request by 1ed**
| Q             | A
| ------------- | ---
| Branch?       | 6.2
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

**Before**

Screenshot from 2023-02-15 23-03-29

**After**

Screenshot from 2023-02-15 23-04-05

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig [hunk 1]
```diff
@@ -90,8 +90,8 @@
                             {% for event in collector.events.events(transport) %}
                                 <tr class="mailer-email-summary-table-row {{ loop.first ? 'active' }}" data-target="#email-{{ loop.index }}">
                                     <td>{{ loop.index }}</td>
-                                    <td>{{ event.message.headers.get('subject').bodyAsString() ?? '(No subject)' }}</td>
-                                    <td>{{ (event.message.headers.get('to').bodyAsString() ?? '(empty)')|replace({'To:': ''}) }}</td>
+                                    <td>{{ event.message.getSubject() ?? '(No subject)' }}</td>
+                                    <td>{{ event.message.getTo()|map(addr => addr.toString())|join(', ')|default('(empty)')|replace({'To:': ''}) }}</td>
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Do we still need this `|replace({'To:': ''})` if we don't use the header string anymore ?

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig [hunk 2]
```diff
@@ -137,12 +137,12 @@
...
                             <p class="mailer-message-subject">
-                                {{ message.headers.get('subject').bodyAsString() ?? '(No subject)' }}
+                                {{ message.getSubject() ?? '(No subject)' }}
                             </p>
                             <div class="mailer-message-headers">
-                                <p><strong>From:</strong> {{ (message.headers.get('from').bodyAsString() ?? '(empty)')|replace({'From:': ''}) }}</p>
-                                <p><strong>To:</strong> {{ (message.headers.get('to').bodyAsString() ?? '(empty)')|replace({'To:': ''}) }}</p>
-                                {% for header in message.headers.all|filter(header => (header.name ?? '') not in ['Subject', 'From', 'To']) %}
+                                <p><strong>From:</strong> {{ message.getFrom()|map(addr => addr.toString())|join(', ')|default('(empty)')|replace({'From:': ''}) }}</p>
+                                <p><strong>To:</strong> {{ message.getTo()|map(addr => addr.toString())|join(', ')|default('(empty)')|replace({'To:': ''}) }}</p>
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Do we still need those replacement if we don't use the header string anymore ?

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig [hunk 3]
```diff
@@ -183,27 +183,27 @@
                                     {% if message.htmlBody %}
                                         {% set htmlBody = message.htmlBody() %}
                                         <div class="tab">
-                                            <h3 class="tab-title">HTML content</h3>
+                                            <h3 class="tab-title">HTML preview</h3>
                                             <div class="tab-content">
                                                 <pre class="prewrap" style="max-height: 600px">
```
Comment (stof) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: This `<pre>` around the iframe does not make sense to me for the HTML preview tab.

**Review by stof — APPROVED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig (hunk 1 above)
Comment (1ed) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Nice catch, thanks!

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig (hunk 3 above)
Comment (1ed) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig]: Removed

**Comment by 1ed** [comment]
Should I fix the encoding stuff in 5.4 too (the tabs are different, so I think that's ok there)? It will cause conflicts, because the redesign.

**Comment by fabpot** [comment]
@1ed Having a PR for 5.4 would be great.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @1ed.

**Pull request merged**